harvard-lts/dais-notifier
# K8s Prep

**K8s prep - Console logging SSL removal, security vulnerabilities, healthchecks*
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/LTSK8S-534

# How should this be tested?

1. Get a copy of the latest .env from Dee Dee
2. Spin up the local instance and run the tests from the [README](https://github.com/harvard-lts/dais-notifier/tree/LTSK8S-570?tab=readme-ov-file#local-setup)
3. Verify http://localhost:10584/readiness displays 'Transfer Service worker ready' after startup

To test logging:
1. In the .env, set `CONSOLE_LOGGING_ONLY=false`
2. Spin up the container `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
3. Verify no logs appear under the logs directory
4. Verify `docker logs notifier` displays logging info
5. Bring down the container `docker-compose -f docker-compose-local.yml down`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no
